packages/web/src/scss/components/Accordion/_Accordion.scss

```diff
@@ -0,0 +1,90 @@
+@use 'sass:map';
+@use 'theme';
+@use '../../tools/typography';
+
+.Accordion__itemHeader {
+    position: relative;
+    display: flex;
+    gap: theme.$accordion-header-gap;
+    align-items: flex-start;
+    justify-content: space-between;
+    width: 100%;
+    height: auto;
+    padding: theme.$accordion-header-padding-y theme.$accordion-header-padding-x;
+    margin-bottom: 0;
+    border-radius: theme.$accordion-border-radius;
+
+    &:hover {
+        background-color: theme.$accordion-item-color-hover;
+    }
+
+    &::before {
+        content: '';
+        position: absolute;
+        top: 0;
+        left: theme.$accordion-header-padding-y;
+        display: block;
+        width: calc(100% - (#{theme.$accordion-header-padding-y} * 2));
+        height: 0;
+        border-bottom: theme.$accordion-divider-width theme.$accordion-divider-style theme.$accordion-item-color-hover;
+    }
+
+    &:hover::before {
+        border-bottom-color: transparent;
+    }
+}
+
+.Accordion__itemToggle {
+    @include typography.generate(theme.$accordion-header-typography);
+
+    display: inline-flex;
+    text-align: left;
+    border: none;
+    background: none;
+    cursor: pointer;
+
+    &:first-of-type::before {
+        content: '';
+        position: absolute;
+        z-index: 0;
+        display: block;
+        inset: 0;
+        width: 100%;
+        height: 100%;
+        border-radius: theme.$accordion-border-radius;
+    }
+
+    &[aria-expanded='true'] {
+        @include typography.generate(theme.$accordion-header-typography-active);
+    }
+
+    // stylelint-disable selector-max-class -- Because of nested element, why this is problem anyway?
+    &[aria-expanded='true'] + .Accordion__itemSlot .Accordion__itemIcon {
+        transform: rotate(180deg);
+    }
+
+    &:active:first-of-type::before {
+        background-color: theme.$accordion-item-color-active;
+    }
+}
+
+.Accordion__itemSlot {
+    display: flex;
+    gap: theme.$accordion-header-gap;
+    align-items: center;
+    justify-content: space-between;
+    width: auto;
+    height: auto;
+}
+
+.Accordion__itemSlot a,
+.Accordion__itemSlot button {
+    position: relative;
+}
+
+.Accordion__item .Collapse__content::after {
```

Comment:
This is still a problem. We cannot style `.Collapse` here. Let's discuss a way to avoid it.